sympy/functions/elementary/hyperbolic.py

```diff
@@ -1702,6 +1750,25 @@ def inverse(self, argindex=1):
     def _eval_rewrite_as_log(self, arg, **kwargs):
         return log(1/arg + sqrt(1/arg - 1) * sqrt(1/arg + 1))
 
+    def _eval_rewrite_as_acosh(self, arg, **kwargs):
+        return sqrt(1/arg - 1)/sqrt(1 - 1/arg)*acosh(1/arg)
```

Comment:
This doesn't always work:
```python
In [1]: f = asech(x)

In [2]: f.subs(x, -1+I).n()
Out[2]: 0.530637530952518 - 2.02307477394609⋅ⅈ

In [3]: f.rewrite(acosh).subs(x, -1+I).n()
Out[3]: -2.02307477394609 - 0.530637530952518⋅ⅈ
```
(a regression compared to master)

Reply:
I think that `return acosh(1/arg)` is good here.

Reply:
Yes, I'll change to that. I cannot even find the earlier rewrite anymore...